### DISCUSSION TITLE
Fail to fit when adding x-ray photometry

### DISCUSSION THREAD

**Discussion by ChengLinLiao** [mboquien/cigale#27] [Q&A]
Hi all,

I add three x-ray photometry in the original photometry sets from optical to radio, but it turns out that X-CIGALE fail to fit. The output from the terminal shows

_Sanity check of the analysis results...
No suitable model found for AS2COS0066.1. It may be that models are older than the universe or that your χ² are very large.
0.0% of the objects have χ²_red~0 and 0.0% χ²_red<0.5_

I've checked the output files, and there are no any value in the “best_model.fits” file. The best-fit age of oldest stars is only 500 Myr, shorter than the age of Universe, and the reduced_chi_square is a null value from the fitting. How could I move on and fix the problem?

Another question is that why some x-ray pivot wavelengths in the filter list are zeros?

Thanks in advance,
Cheng-Lin Liao

**Comment by mboquien** [comment]
@Guang91

**Comment by Guang91** [comment]
Could you share your input data and pcigale.ini file?

The pivotal wavelength *displays* as 0 because of rounding. It's actually
not 0.

**Reply by ChengLinLiao** [reply]
Thanks @Guang91 .
I've solved the problem, and I found that is because I forgot to include the xray module.
Now X-CIGALE runs successfully.

**Comment by Guang91** [comment]
Okay, glad you solved it.
Let me know if you have further questions.

**Reply by ChengLinLiao** [reply]
No other questions. Thanks.
